components.md

```diff
@@ -166,3 +166,29 @@ See `/transformers/optimus-prime.ts` to see how it is done.
 > For Content Transformers to work, we need to add them as modules to Nuxt.
 > Nuxt does this automatically if the nuxt modules are under `/modules`. You
 > can see how to add them as modules in `/modules/transform-module.ts`.
+
+## Data Fetching
+
+In our data fetching processes, we utilize Nuxt's operations such as `$fetch`,
+`useFetch`, and `useAsyncData`. To streamline the management and readability of
+tasks like specifying URLs, setting headers, and defining options in such
+operations, we prefer keeping them in a separate area. In Nuxt examples and as
+it is automatically imported, we place them under the `composables` folder.
+
+- `$fetch`: We use `$fetch` to perform simple API calls.
+- `useAsyncData`: Typically, we make requests with `$fetch` and `useAsyncData`
+  to make them more SSR-friendly while taking advantage of capabilities like
+  watching for changes with the watch parameter.
+- `useFetch`: We using the `useFetch` composable when there is a need to utilize
+  both `$fetch` and `useAsyncData` in the same context. This choice is driven by
+  the desire for a more straightforward data-fetching approach, streamlining the
+  process and enhancing clarity in our codebase.
+
+> :information_source:
+>
+> `useFetch` composable is a wrapper around the `useAsyncData` composable and
+> `$fetch` utility.
+
+See `/composables/github-api.js` to see an example send requests.
```

Comment:
I think "See `/composables/github-api.js`, for example send requests" would be better